content/en/observability_pipelines/monitoring.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Monitoring
+kind: documentation
+disable_toc: false
+---
+
+## Overview
+
+In [Observability Pipelines][1], your pipelines are comprised of components that collect, process, and route your observability data. The health of your pipelines and components are indicated by health statuses and graphs, as well as resource utilization and data delivery graphs.
+
+Health statuses are determined by specific metrics based on thresholds and default time windows. The available statuses are as follows:
+
+- `Healthy`: Indicates the Worker is not falling behind.
+- `Warning`: Indicates the Worker is not performing optimally and is at risk of falling behind. The Worker may fall behind due to issues such as a downstream destination or service causing back pressure to build up and there are not enough resources being provisioned for the Workers.
+- `Critical`: Indicates that the Worker is falling behind. If the Worker is falling behind, it may be at the risk of dropping data; however, so long as your pipelines are [architected][2] and configured correctly, the Worker does not drop data unintentionally.
```

Comment:
```suggestion
- `Critical`: Indicates that the Worker is falling behind. If the Worker is falling behind, it may be at risk of dropping data; however, the Worker will not drop data unintentionally as long as your pipelines are [architected][2] and configured correctly.
```